buildSrc/build.gradle.kts

```diff
@@ -1,5 +1,6 @@
 plugins {
     `kotlin-dsl`
+    id("org.gradle.kotlin-dsl.ktlint-convention") version "0.2.0"
```

Comment:
Is there some documentation on this?

Reply:
See https://github.com/gradle/kotlin-dsl-conventions